ksql-cli/src/test/java/io/confluent/ksql/cli/CliTest.java

```diff
@@ -262,7 +262,7 @@ private void testCreateStreamAsSelect(
     assertRunCommand(
         queryString,
         anyOf(
-            isRow(containsString("Stream " + streamName.toUpperCase() + " created and running")),
+            isRow(containsString("Created query with ID CSAS_" + streamName)),
```

Comment:
If you've changed the text output when a stream is created, you should probably update the docs in the repo to match.